src/main/resources/jenkins/plugins/office365connector/Webhook/config.jelly

```diff
@@ -2,10 +2,10 @@
 <j:jelly xmlns:j="jelly:core"
          xmlns:f="/lib/form">
     <f:entry title="URL" field="url">
-        <f:textbox/>
+        <f:textbox default="${descriptor.url}"/>
```

Comment:
I have question/concern about this value. It is going to be saved with job configuration. Then then you update global value the one for build won't be updated.
Is it good, is it side effect ?

Reply:
It is as we would like it to be. The global settings is meant to function as a default value when creating a job. This to facilitate for example, when you work with a Jenkinsfile and do not want to deal with specifying the URL but use whatever URL has been set globally for that Jenkins instance. I have updated this information in the README-file.